src/gpl/test/simple01-td-tune.ok

```diff
@@ -42,114 +42,127 @@
 ---------------------------------------------------------------------
-[INFO RSZ-0039] Resized 31 instances.
-[INFO GPL-0106] Timing-driven: worst slack 1.42e-09
+[INFO RSZ-0036] Found 1 capacitance violations.
+[INFO RSZ-0039] Resized 74 instances.
+[INFO RSZ-0038] Inserted 1 buffers in 1 nets.
+[INFO GPL-0106] Timing-driven: worst slack 1.41e-09
 [INFO GPL-0103] Timing-driven: weighted 35 nets.
-[NesterovSolve] Iter:   10 overflow: 0.730 HPWL: 4005677
-[NesterovSolve] Iter:   20 overflow: 0.733 HPWL: 3997290
-[NesterovSolve] Iter:   30 overflow: 0.733 HPWL: 3993695
-[NesterovSolve] Iter:   40 overflow: 0.733 HPWL: 3993836
-[NesterovSolve] Iter:   50 overflow: 0.733 HPWL: 3994417
-[NesterovSolve] Iter:   60 overflow: 0.733 HPWL: 3994620
-[NesterovSolve] Iter:   70 overflow: 0.733 HPWL: 3994826
-[NesterovSolve] Iter:   80 overflow: 0.733 HPWL: 3995249
-[NesterovSolve] Iter:   90 overflow: 0.733 HPWL: 3995756
-[NesterovSolve] Iter:  100 overflow: 0.732 HPWL: 3996951
-[NesterovSolve] Iter:  110 overflow: 0.732 HPWL: 3998841
-[NesterovSolve] Iter:  120 overflow: 0.731 HPWL: 4001676
-[NesterovSolve] Iter:  130 overflow: 0.730 HPWL: 4006153
-[NesterovSolve] Iter:  140 overflow: 0.727 HPWL: 4013202
-[NesterovSolve] Iter:  150 overflow: 0.723 HPWL: 4022941
-[NesterovSolve] Iter:  160 overflow: 0.715 HPWL: 4037658
-[NesterovSolve] Iter:  170 overflow: 0.704 HPWL: 4057804
+[NesterovSolve] Iter:   10 overflow: 0.730 HPWL: 4017832
+[NesterovSolve] Iter:   20 overflow: 0.731 HPWL: 4004608
+[NesterovSolve] Iter:   30 overflow: 0.732 HPWL: 4000735
+[NesterovSolve] Iter:   40 overflow: 0.732 HPWL: 4000984
+[NesterovSolve] Iter:   50 overflow: 0.732 HPWL: 4001451
+[NesterovSolve] Iter:   60 overflow: 0.732 HPWL: 4001736
+[NesterovSolve] Iter:   70 overflow: 0.732 HPWL: 4001998
+[NesterovSolve] Iter:   80 overflow: 0.732 HPWL: 4002375
+[NesterovSolve] Iter:   90 overflow: 0.731 HPWL: 4002935
+[NesterovSolve] Iter:  100 overflow: 0.731 HPWL: 4004268
+[NesterovSolve] Iter:  110 overflow: 0.731 HPWL: 4006329
+[NesterovSolve] Iter:  120 overflow: 0.730 HPWL: 4009662
+[NesterovSolve] Iter:  130 overflow: 0.728 HPWL: 4014666
+[NesterovSolve] Iter:  140 overflow: 0.726 HPWL: 4022065
+[NesterovSolve] Iter:  150 overflow: 0.722 HPWL: 4032598
+[NesterovSolve] Iter:  160 overflow: 0.715 HPWL: 4048322
+[NesterovSolve] Iter:  170 overflow: 0.705 HPWL: 4070719
 [INFO GPL-0100] Timing-driven iteration 2/7, virtual: true.
-[INFO GPL-0101]    Iter: 177, overflow: 0.695, keep rsz at: 0.3, HPWL: 4076876
+[INFO GPL-0101]    Iter: 177, overflow: 0.694, keep rsz at: 0.3, HPWL: 4089789
 Iteration |   Area    | Resized | Buffers | Nets repaired | Remaining
 ---------------------------------------------------------------------
         0 |     +0.0% |       0 |       0 |             0 |       356
-    final |    -16.3% |      29 |       0 |             0 |         0
+    final |   -114.9% |      75 |       1 |             1 |         0
```

Comment:
Are these large negative values expected?  Can they really be < 100%?

Reply:
I'll investigate

Reply:
@povik I found the bug here, I'll PR a fix soon

Reply:
yes, the number reported is actually 10x what it should be

Reply:
Unfortunately you'll have to rebase the test output after merging